pkgs/by-name/te/tenv/package.nix

```diff
@@ -38,9 +38,9 @@ buildGoModule rec {
 
   meta = {
     changelog = "https://github.com/tofuutils/tenv/releases/tag/v${version}";
-    description = "A version manager for OpenTofu, Terraform and Terragrunt written in Go";
+    description = "OpenTofu, Terraform, Terragrunt and Atmos version manager, written in Go.";
```

Comment:
What is the point of changing this, besides adding an Atmos?

Reply:
It's fine, but the final period should be removed, see https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

Reply:
done